### DISCUSSION TITLE
Workshop suggestions

### DISCUSSION THREAD

**Discussion by ejh243** [coding-for-reproducible-research/CfRR_Courses#84] [Ideas]
Use this discussion thread to suggest new courses.

**Comment by ejh243** [comment]
How to use Rmarkdown

**Comment by ejh243** [comment]
Also how to use Jupyter notebooks for Python.

**Comment by ejh243** [comment]
How to use an IDE e.g VSCode 
Intermediat Rstudio or RStudio & Git

**Comment by ejh243** [comment]
How to turn code in a package. R/Python.

**Reply by ejh243** [reply]
Perhaps online course with notes.

**Comment by liamjberrisford** [comment]
From the GitHub Issue #93 moved here to bring all of the discussion regarding new courses into a single place. 

Proposed additional course for UKBiobank/population scale data primer course:

The increase in availability of large population scale datasets such as UKBiobank and AllOfUs has highlighted a skills gap in the efficient manipulations and use of these data. it would be great to be able to offer a primer to introduce people to using these resources. The course could be timed to coincide with either the genomics masters or some other PG intake to make this an even more valuable resource.

**Comment by liamjberrisford** [comment]
From the GitHub Issue #90 moved here to bring all of the discussion regarding new course content into one place. 

The introductory R course has a section on plotting in base R, but there is no course that includes plotting with ggplot2 despite most people using it for their final plots.
